akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/scaladsl/ClusterSharding.scala

```diff
@@ -228,16 +198,105 @@ trait ClusterSharding extends Extension { javadslSelf: javadsl.ClusterSharding 
   @InternalApi private[akka] def asJava: javadsl.ClusterSharding = javadslSelf
 }
 
+object ShardedEntity {
+
+  /**
+   * Defines how the entity should be created. Used in [[ClusterSharding#start]]. More optional
+   * settings can be defined using the `with` methods of the returned [[ShardedEntity]].
+   *
+   * @param create Create the behavior for an entity given an entityId
+   * @param typeKey A key that uniquely identifies the type of entity in this cluster
+   * @param stopMessage Message sent to an entity to tell it to stop, e.g. when rebalanced or passivated.
+   *
+   * @tparam A The type of command the entity accepts
```

Comment:
No type paramater `A` anymore, also, since you changed it to `M` should we refer to it as a message in the docs rather than a `command` everywhere?